src/main/scala/viper/gobra/frontend/info/base/BuiltInMemberTag.scala

```diff
@@ -168,12 +177,15 @@ object BuiltInMemberTag {
     override def isPure: Boolean = false
 
     override def typ(config: Config): AbstractType = channelReceiverType(allDirections, c => {
-      val bufferSizeArgType = IntT(config.typeBounds.Int)
-      val sendGivenPermArgType = PredT(Vector(c.elem))
-      val sendGotPermArgType = PredT(Vector()) // pred() because we enforce that sendGotPermArgType == recvGivenPermArgType
-      val recvGivenPermArgType = PredT(Vector())
-      val recvGotPermArgType = PredT(Vector(c.elem))
-      FunctionT(Vector(bufferSizeArgType, sendGivenPermArgType, sendGotPermArgType, recvGivenPermArgType, recvGotPermArgType), VoidType)
+      // init's signature is adapted to the heavy simplifications that are in place for the initial support for channels.
+      // in particular, the permission for SendGivenPerm and RecvGotPerm as well as SendGotPerm and RecvGivenPerm have
+      // to be equal. Thus, they get merge two parameters: The former pair is called `proPerm` as they describe the
```

Comment:
minor thing: "get merge" in line 182 sounds odd. Besides, I would rename `proPerm` to `sendPerm` as the permissions flow in the direction of the send operation and I would rename "contraPerm" to "recvPerm"

Reply:
I was thinking of calling it `sendPerm` but then opted for a completely different name as one might think that `sendPerm` are `SendGivenPerm` and `SendGotPerm` which is not the case